src/core/include/mp-units/math.h

```diff
@@ -144,10 +145,24 @@ template<auto R, typename Rep>
 }
 
 /**
- * @brief Determines if a number is infinite.
+ * @brief Determines if a quantity point is finite.
  *
- * @param a: Number to analyze.
- * @return bool: Whether the number is infinite or not.
+ * @param a: Quantity point to analyze.
+ * @return bool: Whether the quantity point is finite or not.
+ */
+template<auto R, auto PO, typename Rep>
+  requires requires(Rep v) { isfinite(v); } || requires(Rep v) { std::isfinite(v); }
+[[nodiscard]] constexpr bool isfinite(const quantity_point<R, PO, Rep>& a) noexcept
+{
+  using std::isfinite;
+  return isfinite(a.quantity_ref_from(a.point_origin).numerical_value_ref_in(a.unit));
```

Comment:
Just reuse the other overload:

```suggestion
  return isfinite(a.quantity_ref_from(a.point_origin));
```

Please do the same for the rest